### PR TITLE
add index page

### PR DESCRIPTION
**Pull request by wahyuoi**
this PR to resolve page index issue on /tasks/tools
image

/cc @phanama 
/assign @girikuncoro

### CODE REVIEW

File: content/id/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Instalasi Alat"
```
Comment (phanama) [inline content/id/docs/tasks/tools/_index.md:2]:
```suggestion
title: "Instalasi Perkakas"
```

Comment (phanama) [reply content/id/docs/tasks/tools/_index.md:2]: How about using "Perkakas"?

https://kbbi.kemdikbud.go.id/entri/perkakas

Comment (phanama) [reply content/id/docs/tasks/tools/_index.md:2]: Ah, just checked RSNI and it uses "alat".
I feel like "peralatan" or "perkakas" fits more though. Any of them is ok, at your choice.

Comment (wahyuoi) [reply content/id/docs/tasks/tools/_index.md:2]: i'll pick `peralatan`

File: content/id/docs/tasks/tools/_index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Instalasi Peralatan"
```
Comment (phanama) [inline content/id/docs/tasks/tools/_index.md:2]:
```suggestion
title: "Menginstal Peralatan"
```

Comment (phanama) [reply content/id/docs/tasks/tools/_index.md:2]: Skipped this. It should be a verb.

Comment (wahyuoi) [reply content/id/docs/tasks/tools/_index.md]: fixed in [4886c64](https://github.com/kubernetes/website/commit/4886c6443e7d06bf85252c47527bf9dc5471e737)

**Comment by girikuncoro** [comment]
Thanks for this index page @wahyuoi ! 🎉 

/lgtm
/approve

**Pull request merged**